GameData/002_CommunityPartsTitles/Localization/CPT_Mk2Expansion_loc.cfg

```diff
@@ -7,42 +7,41 @@
 @PART[adapterSize2-Mk2]:NEEDS[Mk2Expansion]    {@title = #CPT_adapterSize2-Mk2  }  // stock need new title Mk2-25 Adapter-C       // CPT = Mk2-25 Adapter        // stock =  2.5m to Mk2 Adapter
 
 
-
-
 Localization
 {
 	en-us
 	{
-		//Command
-		#LOC_M2X_Angler_title = Mk2 Multipurpose Cockpit         // SC-TD Multipurpose Cockpit
-		#LOC_M2X_Blade_title  = Mk2 Hypersonic Cockpit           // HS-X Hypersonic Cockpit
-		#LOC_M2X_Tuna_title   = Mk2 'Fishhead' Cockpit           // HV Mk2-1 Cockpit
+		//  Command
+		#LOC_M2X_Angler_title = Mk2 Cockpit (Multipurpose)       // SC-TD Multipurpose Cockpit
+		#LOC_M2X_Blade_title  = Mk2 Cockpit (Hypersonic)         // HS-X Hypersonic Cockpit
+		#LOC_M2X_Tuna_title   = Mk2 Cockpit 'Fishhead'           // HV Mk2-1 Cockpit
+		#LOC_M2X_Viper_title  = Mk2 Cockpit 'Viper'              // CB-C Cockpit (3 variants)
 		#LOC_M2X_Raven_title  = Mk2-Mk1 Cockpit (Inline)         // R-71 Mk2 Cockpit
-		#LOC_M2X_Viper_title  = Mk2 'Viper' Cockpit (3 variants) // CB-C Cockpit (3 variants)
 
 
 		// Aero
 		#LOC_M2X_CMCap_title         = Chine Mk1 (A-Cap)
-		#LOC_M2X_CMLong_title        = Chine Mk1 (D-Long)
 		#LOC_M2X_CMShort_title       = Chine Mk1 (B-Short)
-		#LOC_M2X_ShortChine_title    = Chine Mk2 Segment-B (Configurable)    // Comfigurable Short Chine Segment
-		#LOC_M2X_StandardChine_title = Chine Mk2 Segment-C (Configurable)    // Comfigurable Chine Segment
-		#LOC_M2X_longChine_title     = Chine Mk2 Segment-D (Configurable)    // Comfigurable Long Chine Segment
+		#LOC_M2X_CMLong_title        = Chine Mk1 (C-Long)
```

Comment:
idea there is C - always "Normal", 
so Short is B, Long is D, Very Short is A, and very Long is E.
Sometimes there is no C, so it A, B, and D.